src/apps/activities/tests/fixtures/scores_reports.py

```diff
@@ -90,6 +99,17 @@ def subscale(subscale_item: SubscaleItem) -> Subscale:
     )
 
 
+@pytest.fixture
+def scores_and_reports_lookup_scores(score: Score, section: Section, subscale: Subscale) -> ScoresAndReports:
+    return ScoresAndReports(
+        generate_report=True,
+        show_score_summary=True,
+        reports=[score, section],
+        scoring_type="lookup_scores",
+        subscale_name=subscale.name,
```

Comment:
Move these properties to the score object, and change the value of `scoring_type` to `score`